Doc/library/os.rst

```diff
@@ -1195,6 +1195,32 @@ or `the MSDN <https://msdn.microsoft.com/en-us/library/z0kc8e3z.aspx>`_ on Windo
    .. versionadded:: 3.3
 
 
+.. function:: preadv(fd, buffers, offset, flags = None)
+
+   Combines the functionality of :func:`os.readv` and :func:`os.pread`. It
+   performs the same task as :func:`os.readv`, but adds a fourth argument,
+   offset, which specifies the file offset at which the input operation is
```

Comment:
You put asterisks around flags below and I think the first offset should have the name because this is referring to the argument.

Reply:
Corrected in 1988d1a